contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -9,6 +9,17 @@ Feature gates are intended to cover the development life cycle of a feature - th
 Features generally progress through `Alpha` -> `Beta` -> `GA`. Sometimes we end up deciding that a feature is not going to be supported and we end up marking them as `Deprecated`.
 
 The majority of features will go through all three stages, but occasionally there are features which may skip stages.
+While some exceptions may happen, approvers should use the following guidance:
+- features that involve API changes must progress through all `Alpha`, `Beta`, `GA` stages
+- larger or more complex changes should progress through all `Alpha`, `Beta`, `GA` stages
+- smaller changes that carry non-trivial risk (e.g. due to changing user-facing behavior) might skip
```

Comment:
```suggestion
- features with minimal complexity but that still carry non-trivial risk (e.g. due to changing user-facing behavior)  or changes to  the performance or scalability characteristics of a cluster might skip
```

Reply:
+1 to the clarification.

We should also indicate that changes that carry a risk of making previously working functionality no longer work in certain edges should always start in an off-by-default state.

Reply:
done